pkg/test/resourcebuilder/control_plane_machine_set.go

```diff
@@ -23,17 +23,29 @@ import (
 )
 
 const (
-	// controlPlaneMachineSetName is the only valid name allowed.
+	// ControlPlaneMachineSetName is the only valid name allowed.
 	// A ControlPlaneMachineSet is a singleton within the cluster, this matches other singletons such as Infrastructure.
-	controlPlaneMachineSetName = "cluster"
+	ControlPlaneMachineSetName = "cluster"
 )
 
+type CPMS interface {
```

Comment:
Acronyms are bad, lets spell this out and call it (common pattern from kube). Make sure to add godocs in this file too
```suggestion
type ControlPlaneMachineSetInterface interface {
```